src/sql/workbench/contrib/backup/browser/backup.component.ts

```diff
@@ -528,19 +537,22 @@ export class BackupComponent extends AngularDisposable {
 			this.recoveryBox!.disable();
 			this.mediaNameBox!.disable();
 			this.mediaDescriptionBox!.disable();
-			if (this._engineEdition === DatabaseEngineEdition.SqlManagedInstance) {
+			if (this._allowFileBackup && this._allowUrlBackup) {
+				this.toUrlCheckBox.enable();
+			}
+			if (this._allowUrlBackup) {
 				this.toUrlCheckBox.checked = true;
```

Comment:
Would this mean on-prem servers would open the dialog in URL-only mode by default?